packages/shorebird_tests/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: shorebird_tests
+description: Shorebird's  Flutter customizations tests
+version: 1.0.0
+
+environment:
+  sdk: ^3.3.4
+
+dependencies:
+  archive: ^3.5.1
+  path: ^1.9.0
```

Comment:
Do we need these dependencies?

Reply:
Yeah,

path is used here for example: <https://github.com/shorebirdtech/flutter/pull/45/files#diff-f89684ff9eb7b28bb4625302b31d2c083d33414ed9bde6331f7991098896b412R15>
and archive is used here: <https://github.com/shorebirdtech/flutter/pull/45/files#diff-f89684ff9eb7b28bb4625302b31d2c083d33414ed9bde6331f7991098896b412R189>